doc/src/tutorial/gotchas.rst

```diff
@@ -266,7 +266,7 @@ division, depending on whether you are in Python 2 or Python 3, and depending
 on whether or not you have run ``from __future__ import division``:
```

Comment:
This should be rewritten as ``from __future__ import division`` is Python 2 which is no longer supported.

Reply:
@oscargus How about I mention the `SymPy` version too from where Python 2 is not supported like:
``from __future__ import division`` for Python 2 which is no longer supported for versions above `SymPy 1.5.1`